src/binding.cc

```diff
@@ -3148,3 +3149,22 @@ void v8__CompiledWasmModule__DELETE(v8::CompiledWasmModule* self) {
   delete self;
 }
 }  // extern "C"
+
+// icu
+
+extern "C" {
+
+void icu_get_default_locale(char* output) {
+  const icu_71::Locale& default_locale = icu::Locale::getDefault();
+  UErrorCode status = U_ZERO_ERROR;
+  auto tag = default_locale.toLanguageTag<std::string>(status);
+  std::copy(tag.begin(), tag.end(), output);
+  output[tag.size()] = '\0';
```

Comment:
This seems a little dangerous. Can you add an assert that tag.length is less than 1024 - 1?